Dockerfile

```diff
@@ -1,13 +1,14 @@
-FROM alpine:3.4
+FROM alpine:3.6
 
 ENV GOPATH /go
 
-RUN apk update && apk add ca-certificates go git && \
+RUN apk update && apk add ca-certificates go git musl-dev && \
     rm -rf /var/cache/apk/* && \
     go get -u github.com/xenolf/lego && \
     cd /go/src/github.com/xenolf/lego && \
+    git checkout tags/v0.4.0 && \
```

Comment:
I would also add to beginning 
`LEGO_VERSION tags/v0.4.0`

and then change to:
`git checkout ${LEGO_VERSION}`